.eslintrc

```diff
@@ -96,6 +97,23 @@ rules:
   function-paren-newline: [0]
   generator-star-spacing: [0]
   getter-return: [2]
+  github/array-foreach: [2]
```

Comment:
Hmm, what's the reasoning behind this one?

Reply:
See https://github.com/github/eslint-plugin-github/blob/main/docs/rules/array-foreach.md#rule-details. I personally prefer `for-of` because it allows flow control statements like `return` or `break` in the body, something which isn't possible in a function like `forEach`.

Reply:
Seems like in this context it's fine and nice to enable the rule.